### PR TITLE
Add handler for offset increments

### PR DESCRIPTION
**Pull request by rsmichaeldunn**
This PR adds handling of simplified offset increments to cronex.  Offset increments are where you want a cron job to run every n periods, but _not_ starting at period 0.  For instance, if you wanted a job to start every 15 minutes, at xx:05, xx:20, xx:35, and xx:50, the cron expression would look like `5/15 * * * *`.

Cronex already handles this via the more verbose `5-59/15 * * * *` notation.  This PR would allow that to be simplified and aligns with the Quartz Scheduler implementation of cron (which is what most online cron expression generator tools use)

### CODE REVIEW

File: cronex/__init__.py [hunk 1]
```diff
@@ -341,6 +348,9 @@ def parse_atom(parse, minmax):
             prefix, suffix = [int(n) for n in subrange.split('-')]
             if prefix < minmax[0] or suffix > minmax[1]:
                 raise ValueError("\"%s\" is not within valid range." % parse)
+        elif string_is_int(subrange):
```
Comment (ericpruitt) [inline cronex/__init__.py]: Is there any reason `subrange.isdigit()` wouldn't work here?

Comment (rsmichaeldunn) [reply cronex/__init__.py]: Not that I can think of.  Updated.

**Comment by rsmichaeldunn** [comment]
Yikes, my venv files got committed...  one moment while I fix that.

**Comment by rsmichaeldunn** [comment]
There.  I think it's ready now.

**Comment by ericpruitt** [comment]
This has been committed as https://github.com/ericpruitt/cronex/commit/108ac1adfd3deac41483f8832cbb5ad9392b0d2d and released as version 0.1.3 on PyPI. Thanks for the contribution.

**Pull request closed without merging**